### PR TITLE
Azure install guide

### PR DESCRIPTION
**Pull request by egernst**

### CODE REVIEW

**Comment by egernst** [comment]
@klynnrif @intelkevinputnam can you PTAL?  

This is a basic placeholder which will likely be augmented later.

File: install/azure-installation-guide.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Install Kata Containers on Azure
+
+Kata Containers on Azure use nested virtualization to provide an identical installation
+experience to Kata on your preferred Linux distribution. 
+
+This guide assumes you have an Azure account set up and tools to remote into your virtual
```
Comment (jodh-intel) [inline install/azure-installation-guide.md]: Nit: "... to remotely login to your..." would be clearer I think.

File: install/README.md [hunk 2]
```diff
@@ -84,6 +84,7 @@ Manual installation instructions are available for [these distributions](#suppor
 
 ## Installing on a Cloud Service Platform
 * [Amazon Web Services (AWS)](aws-installation-guide.md)
+* [Azure](azure-installation-guide.md)
```
Comment (jodh-intel) [inline install/README.md]: Oh wait - shouldn't we say "Microsoft* Azure"?

Comment (egernst) [reply install/README.md]: I'm not sure -- this came from Sarah initially - I'll leave it to her discretion.  @scooley - WDYT?

File: install/azure-installation-guide.md (hunk 1 above)
Comment (egernst) [reply install/azure-installation-guide.md]: done

File: install/README.md (hunk 2 above)
Comment (jodh-intel) [reply install/README.md]: All the other providers are listed, so I think for consistency alone we should specify the providing company name.

Comment (grahamwhaley) [reply install/README.md]: I vote we should add 'Microsoft' in the phrase - that is how it is called on their site, and they no doubt have some trademark or copyrights, so we should ack here.

**Comment by grahamwhaley** [comment]
I've labelled this as WIP, as it obviously will get more updates, and the two commits can be folded as well before merge imo.

**Comment by grahamwhaley** [comment]
have re-read - I'm happy - it is maybe deceptively 'too simple' :-)
merging...

**Pull request merged**